test/Passes/tile-and-fuse-named-op.mlir

```diff
@@ -129,3 +129,34 @@ func.func @matmul_sequence_fusion_with_relu(%arg0: tensor<32x64xf32>, %arg1: ten
 // CHECK-NEXT: }
 
 // -----
+
+func.func @matmul_chain_multi_use_into_relu(%arg0: tensor<32x64xf32>, %arg1: tensor<64x32xf32>,
```

Comment:
Is there any negative case with two users that would prevent fusion?
If not, it'd be nice to add to validate that path too.

Reply:
added a negative test.